config/menu.php

```diff
@@ -1,92 +1,125 @@
-            'blogs' => [
-                'route' => '/content/blog',
-            ],
-            'offers' => [
-                'route' => 'http://vana.trip.ee/reisipakkumised',
-                'external' => true,
-            ],
+    'header' => [
+           'home' => [
+            'route' => '/',
         ],
+        'news' => [
+            'route' => '/content/news',
+        ],
+       'flights' => [
+            'route' => '/content/flight',
+        ],
+        'travelmates' => [
+            'route' => '/content/travelmate',
+        ],
+        'forum' => [
+            'route' => '/content/forum',
+        ],
+        'photos' => [
+            'route' => '/content/photo',
+        ],
+        'blogs' => [
+            'route' => '/content/blog',
+        ],
+        'offers' => [
+            'route' => 'http://vana.trip.ee/reisipakkumised',
+            'external' => true,
+        ],
+    ],
+
+    'footer' => [
+        'news' => [
+            'route' => '/content/news',
+        ],
+        'flights' => [
+            'route' => '/content/flight',
+        ],
+        'offers' => [
+            'route' => 'http://vana.trip.ee/reisipakkumised',
+            'external' => true,
+        ],
+        'travelmates' => [
+            'route' => '/content/travelmate',
+        ],
+    ],
+
+    'footer2' => [
+        'forum' => [
+            'route' => '/content/forum',
+        ],
+        'expat' => [
+            'route' => '/content/expat',
+        ],
+        'buysell' => [
+            'route' => '/content/buysell',
+        ],
+    ],
+
+    'footer3' => [
+        'about' => [
+            'route' => '/content/static/1534',
+        ],
+        'contact' => [
+            'route' => '/content/static/972',
+        ],
+        'eula' => [
+            'route' => '/content/static/25151',
+        ],
+        'advertising' => [
+            'route' => '/content/static/22125',
+        ],
+        'styleguide' => [
+            'route' => '/styleguide',
+        ],
+    ],
 
-        'footer' => [
-            'about' => [
-                'route' => '/content/static/1534',
-            ],
-            'contact' => [
-                'route' => '/content/static/972',
-            ],
-            'eula' => [
-                'route' => '/content/static/25151',
-            ],
-            'advertising' => [
-                'route' => '/content/static/22125',
-            ],
-            'styleguide' => [
-                'route' => '/styleguide',
-            ],
-            'facebook' => [
-                'route' => 'https://facebook.com/tripeeee',
-                'external' => true,
-            ],
-            'twitter' => [
-                'route' => 'https://twitter.com/trip_ee',
-                'external' => true,
-            ],
+    'footer-social' => [
+        'facebook' => [
+            'route' => 'https://facebook.com/tripeeee',
+            'external' => true,
+            'modifier' => 'm-facebook',
```

Comment:
Interesting use case, I'd recommend to use 'icon' property instead but let's discuss. Whaterver the result, follow up on #298